node/cmd/mountns/main.go

```diff
@@ -0,0 +1,47 @@
+//go:build cgo
+
+// Copyright (c) 2022 Tigera, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"os"
+	"syscall"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/projectcalico/calico/felix/bpf"
+)
+
+/*
+#define _GNU_SOURCE
+#include <sched.h>
+#include <fcntl.h>
+
+__attribute__((constructor)) void set_namespaces(void) {
+	setns(open("/initproc/ns/cgroup", O_RDONLY, 0), CLONE_NEWCGROUP);
+	setns(open("/initproc/ns/mnt", O_RDONLY, 0), CLONE_NEWNS);
+} */
+import "C"
+
+func main() {
+	logrus.Info("Trying to mount root cgroup fs.")
```

Comment:
Suggest not even importing logrus/our bpf library here and using `fmt.Println()` for logging.  I think importing things risks:
* bloating the binary
* having some unexpected initialisation logic run that, for example, starts a goroutine you don't want.

Looks like with logrus, the binary is 24MB but remving logrus and the bpf  import, it goes down to 1.3MB.  Seems like a worthwhile saving but would need to comment why we're not using normal frameworks!